contracts/cw-core/src/tests.rs

```diff
@@ -1730,11 +1730,11 @@ fn test_cw721_receive() {
     app.execute_contract(
         Addr::unchecked(CREATOR_ADDR),
         cw721_addr.clone(),
-        &cw721_base::msg::ExecuteMsg::Mint(cw721_base::msg::MintMsg::<Option<Empty>> {
+        &cw721_base::msg::ExecuteMsg::<Empty, Empty>::Mint(cw721_base::msg::MintMsg::<Empty> {
```

Comment:
Confused as to these changes are needed. Are these unrelated items?
Maybe a clean rebase/cherry-pick will remove them if they from some other changes in main

Reply:
Since 0.15 otherwise it fails to compile
```rust
error[E0282]: type annotations needed
    --> contracts/cw-core/src/tests.rs:1733:10
     |
1733 |         &cw721_base::msg::ExecuteMsg::Mint(cw721_base::msg::MintMsg::<Empty> {
     |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ cannot infer type for type parameter `E` declared on the enum `ExecuteMsg`
```
I assume it's due to various recent bumps and macro usage introduction. 
https://github.com/CosmWasm/cw-nfts/commit/2e10714380b41b7a578fd1f18ffe217c29cc4e21#diff-53cbee29b7364a632335866c0cf7ebf8dd07f4b832cb4d5ae78f666bcbb184f6

Reply:
Ah, I missed the cw721 version bump and api changes.
Comment withdrawn